mage_ai/data_preparation/repo_manager.py

```diff
@@ -114,6 +117,10 @@ def from_dict(self, config_dict: Dict) -> 'RepoConfig':
     def metadata_path(self) -> str:
         return os.path.join(self.repo_path, 'metadata.yaml')
 
+    @property
+    def repo_spark_config(self) -> dict:
```

Comment:
Is this method same with self.spark_config?

Reply:
Right, removed this property to avoid duplication.